analyses/mwi/dryspells_trigger/exploration/ecmwf_cds_compare_aggr.md

```diff
@@ -177,18 +179,19 @@ def combine_for_obs(df_for,df_obs):
 ```
 
 ```python
-meth_dict={"unrounded_center":{"use_unrounded_area_coords":True,"resolution":None,"all_touched":False},
-"rounded_center":{"use_unrounded_area_coords":False,"resolution":None,"all_touched":False},
-"rounded_weightavg":{"use_unrounded_area_coords":False,"resolution":0.05,"all_touched":False},
-"rounded_alltouched":{"use_unrounded_area_coords":False,"resolution":None,"all_touched":True},
+meth_dict={"unrounded_center":{"use_unrounded_area_coords":True,"resolution":None,"all_touched":False,"source_cds":True},
+"rounded_center":{"use_unrounded_area_coords":False,"resolution":None,"all_touched":False,"source_cds":True},
+"rounded_weightavg":{"use_unrounded_area_coords":False,"resolution":0.05,"all_touched":False,"source_cds":True},
+"rounded_alltouched":{"use_unrounded_area_coords":False,"resolution":None,"all_touched":True,"source_cds":True},
 }
 ```
 
 Load the forecast data for different methods of aggregation
 
 ```python
 for key, values in meth_dict.items():
-    values["df"]=load_forecast_data(country_iso3,config,values["use_unrounded_area_coords"],resolution=values["resolution"],all_touched=values["all_touched"],
+    values["df"]=load_forecast_data(country_iso3,config,values["use_unrounded_area_coords"],
```

Comment:
I think this needs to be updated to reflect the new filenames (e.g. with the coordinate bounds)